### PR TITLE
Release 14.0.0

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
     },
     {
       "name": "theforeman/foreman",
-      "version_requirement": ">= 10.0.0 < 16.0.0"
+      "version_requirement": ">= 10.0.0 < 17.0.0"
```
[the remark this answers is not in the document]
Comment (wbclark) [reply metadata.json:31]: Thanks, I've opened it here: https://github.com/theforeman/puppet-foreman_proxy/pull/595

[the remark this answers is not in the document]
Comment (ekohl) [reply metadata.json:31]: Thanks. Needs an update now

**Comment by wbclark** [comment]
I've rebased this

**Comment by wbclark** [comment]
Rebased

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## [14.0.0](https://github.com/theforeman/puppet-foreman_proxy/tree/14.0.0) (2020-05-12)
```
Comment (ekohl) [inline CHANGELOG.md:3]:
```suggestion
## [14.0.0](https://github.com/theforeman/puppet-foreman_proxy/tree/14.0.0) (2020-05-15)
```

**Comment by wbclark** [comment]
Rebased and made requested changes

**Pull request merged**